src/js/autocomplete.js

```diff
@@ -167,7 +173,44 @@ class Autocomplete {
 			displayMenu: 'overlay',
 			showNoOptionsFound: false,
 			templates: {
-				suggestion: this.suggestionTemplate.bind(this)
+				/**
+				 * Used when rendering suggestions, the return value of this will be used as the innerHTML for a single suggestion.
+				 * @param {*} option The suggestion to apply the template with.
+				 * @returns {string} HTML string to represent a single suggestion.
+				 */
+				suggestion: (option) => {
+					// If the `optionToSuggestion` function is defined
+					// Apply the function to the option. This is a way for the
+					// consuming application to decide what text should be
+					// shown for this option.
+					// This is usually defined when the option is not already a string.
```

Comment:
Should we error if `option` is not a string and `optionToSuggestion` is not set?

Reply:
Good suggestion 👍 
I've added that in this commit [`c78e6ed` (#12)](https://github.com/Financial-Times/o-autocomplete/pull/12/commits/c78e6edec54b177b2cb8393a6cf4edfb688ef88f)
It was hard to test due to the fact that accessible-autocomplete uses preact to render and preact renders asynchronously, but, it is possible to test :D